src/makeCalculator.js

```diff
@@ -37,7 +37,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(prevNumber, number) {
```

Comment:
will be better to create more general variable names
it's not clear why one of numbers is previous, and other one not